pkgs/applications/blockchains/besu/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "besu";
-  version = "21.10.8";
+  version = "22.4.3";
 
   src = fetchurl {
     url = "https://hyperledger.jfrog.io/artifactory/${pname}-binaries/${pname}/${version}/${pname}-${version}.tar.gz";
```

Comment:
Is there a reason to not use GitHub?
Like: `https://github.com/hyperledger/besu/archive/refs/tags/22.4.3.tar.gz`

Reply:
That is the source whereas the jfrog archive is a binary.

Reply:
Hence #181374